### ISSUE TITLE
Do you have support BEP0044? dht_store 

### ISSUE THREAD

**Issue by hfeeki** [nictuku/dht#49]
Do you have support BEP044? dht_store  , If not , then do you have a plan to support it ? Thanks very much!

**Comment by spikebike** [comment]
I don't believe so.  I'll look into it.  I've become more interested in BEP0044 now that BEP0046 depends on it.  More info at:
https://torrentfreak.com/mutable-torrents-proposal-makes-bittorrent-resilient-160813/
http://www.bittorrent.org/beps/bep_0046.html

What was your use case?
